src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/Usage/DoNotCallOrderByMultipleTimes.cs

```diff
@@ -0,0 +1,85 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the MIT license.  See License.txt in the project root for license information.
+
+using System.Collections.Immutable;
+using Analyzer.Utilities;
+using Analyzer.Utilities.Extensions;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.Diagnostics;
+using Microsoft.CodeAnalysis.Operations;
+
+namespace Microsoft.NetCore.Analyzers.Usage
+{
+    using static MicrosoftNetCoreAnalyzersResources;
+
+    [DiagnosticAnalyzer(LanguageNames.CSharp, LanguageNames.VisualBasic)]
+    public class DoNotCallOrderByMultipleTimes : DiagnosticAnalyzer
+    {
+        internal const string RuleId = "CA2259";
+
+        internal static readonly DiagnosticDescriptor Rule = DiagnosticDescriptorHelper.Create(
+            RuleId,
+            CreateLocalizableResourceString(nameof(DoNotCallOrderByMultipleTimesTitle)),
+            CreateLocalizableResourceString(nameof(DoNotCallOrderByMultipleTimesDescription)),
```

Comment:
This is misleading. The parameter that follows title is a message, not description.

https://github.com/dotnet/roslyn-analyzers/blob/1595b90e6f3c387d684c04b3a570f87bce4e7580/src/Utilities/Compiler/DiagnosticDescriptorHelper.cs#L14-L17

Reply:
I checked various analyzers and they seem to use a message that's identical to the title, so I'm not doing that, too. However, the `sarif` file apparently uses the title for the `shortDescription`, and the message for the `fullDescription`. That's a bit weird.